app/models/server/models/Rooms.js

```diff
@@ -229,6 +229,22 @@ export class Rooms extends Base {
 		return this.update(query, update);
 	}
 
+	setReadOnlyByUserId(_id, readOnly, reactWhenReadOnly) {
+		const query = {
+			uids: _id,
```

Comment:
this will probably make event multiple DM rooms as read only.. which I'm not sure is the desired behavior 🤔

Reply:
Well, it should mark all the direct conversations with the deactivated user as readonly 🤔

Reply:
right, is that behavior expected? like if I'm talking to 5 different people, if one of them gets deactivated, should the whole conversation be locked up? this might be already better than what we have today though =)